lib/connectors/memory.js

```diff
@@ -228,6 +228,15 @@ Memory.prototype.updateOrCreate = function (model, data, callback) {
   });
 };
 
+Memory.prototype.findOrCreate = function (model, query, data, callback) {
+  this.all(model, query, function (err, list) {
+    if (err || (list && list[0])) return callback(err, list && list[0], false);
+    this.create(model, data, function (err) {
+      callback(err, data, true);
+    });
+  }.bind(this));
+};
+
```

Comment:
@bajtos you are right. I'm lazy. This only for testing the case that connector *has* `findOrCreate()`. But now, the tests can't cover the case that connector does NOT have `findOrCreate()`.

I should extend the memory connector and implement `findOrCreate()` in a standalone test.